src/main/java/net/sf/jabref/external/DroppedFileHandler.java

```diff
@@ -129,8 +130,8 @@ public void handleDroppedfile(String fileName, ExternalFileType fileType, MainTa
      *                  to a local temporary file.
      * @param entry     The target entry for the drop.
      */
-    public void handleDroppedfile(String fileName, ExternalFileType fileType, BibEntry entry) {
-        NamedCompound edits = new NamedCompound(Localization.lang("Drop %0", fileType.getExtension()));
+    public void handleDroppedfile(String fileName, Optional<ExternalFileType> fileType, BibEntry entry) {
```

Comment:
I still don't get why an optional argument is necessary here instead of a simple ExternalFileType (which has to be non-null). The method apparently needs the fileType, so the argument is not optional...